src/styles/_Variables.scss

```diff
@@ -7,8 +7,8 @@ $breakpoint-extra-small: 640px;
 $breakpoint-small: 768px;
 $breakpoint-medium: 1024px;
 
-$grid-max-width: 62rem;
-$container-max-width: 102.4rem;
-$container-max-width-sm: 64rem;
+$container-max-width: 1200px;
+
+$content-max-width: 620px;
```

Comment:
`$grid-max-width` and `$container-max-width-sm` were very similar, so I merged them into a single `$content-max-width` variable.